tests/unit/header-cell-test.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import {
+  moduleFor,
+  test
+} from 'ember-qunit';
+
+import ColumnDefinition from '../../models/column-definition';
+
+moduleFor('view:headerCell', 'Unit: Header Cell', {
+  unit: true,
+  needs: [
+    'template:header-cell'
+  ]
+});
+
+test('The elementSizeDidChange scheduled task is correctly canceled when the element is destroyed', function(assert) {
+  assert.expect(3);
+
+  let headerCell = this.subject({
+    content: ColumnDefinition.create({}),
+    tableComponent: {
+      columnMode: 'standard'
+    }
+  });
+
+  Ember.run(function() {
```

Comment:
`this.append` is only defined for tests using `moduleForComponent`, and that only works with actual components, not views. `append` has to be called on the view directly, and must be in a run loop.